src/Umbraco.Infrastructure/Persistence/SqlServerDbProviderFactoryCreator.cs

```diff
@@ -3,12 +3,12 @@
 using System.Linq;
 using Microsoft.Extensions.Options;
 using NPoco;
-using Umbraco.Cms.Core.Composing;
 using Umbraco.Cms.Core.Configuration.Models;
 using Umbraco.Cms.Infrastructure.Persistence.SqlSyntax;
 
 namespace Umbraco.Cms.Infrastructure.Persistence
 {
+    [Obsolete("This is only used for integration tests and should be moved into a test project.")]
```

Comment:
Or remove this class entirely?

Reply:
Obsolete is okay, I fear some packages could use it, so we keep it for v9, and move for v10